ansible_base/rbac/validators.py

```diff
@@ -116,6 +116,16 @@ def validate_permissions_for_model(permissions, content_type: Optional[Model], m
                 display_perms = ', '.join(non_add_model_permissions)
                 raise ValidationError({'permissions': f'Permissions for model {role_model._meta.verbose_name} needs to include view, got: {display_perms}'})
 
+    # check change edit permissions is given for every model that has update/delete/special actions listed
+    for cls, valid_model_permissions in permissions_by_model.items():
+        if 'delete' in cls._meta.default_permissions:
```

Comment:
Nice thinking here. However, I'd like to request that we `and` this with `'change' in cls._meta.default_permissions`. The logic here only makes sense to me when both delete & change are valid permissions for the model type.